setup.py

```diff
@@ -2,10 +2,12 @@
 from setuptools import find_packages
 
 REQUIREMENTS = [
+    "pylzma"
```

Comment:
Please specify major version like so `pylzma==0.*`, just in case the library API change with 1.x.x release

Reply:
Ok, done.

Nothing fancy, just a minor quality of life change 😄 

Apparently it was not working on Windows before, at least for me, even though I had 7z in PATH. This will be helpful as I was planning to use your package for some other stuff.